tests/unit/model/projections/changed-attrs-test.js

```diff
@@ -88,4 +88,191 @@ module('unit/model/projections/changed-attrs', function(hooks) {
       'changed attributes is correct'
     );
   });
+
+  test('Can set a many embedded property to a semi resolved array containing a mix of pojos and megamorphic models', function(assert) {
```

Comment:
let's move this test out of projections; something like `tests/unit/model/changed-attrs` or `tests/unit/model/nested-model`.

I'm not too concerned as long as it's not wrong (eg non-proj test in proj) and as long as we're not creating giant test files (eg `model-test.js`)